src/main/java/frc/robot/Constants.java

```diff
@@ -3,8 +3,16 @@
 
 public final class Constants {
 
+    public static final double NOMINAL_VOLTAGE = 12; // [V]
+
     public static class ExampleSubsystem {
         public static final double POWER = 0.5; // [%]
     }
 
+    public static class Intake {
+        public static final double DEFAULT_POWER = 0.5; // power intake will receive on the basic command. [%]
+        public static final double POWER_TO_VELOCITY = -3 / 16.0; // Ratio of power to velocity. [% / m/s]
+        public static final double BIAS = 1; // Maximal power of the intake. [%]
```

Comment:
Bias is pretty general IMO. How do you feel about `GRAVITY_COMPENSATING_POWER` (IIUC the purpose of this constant)?